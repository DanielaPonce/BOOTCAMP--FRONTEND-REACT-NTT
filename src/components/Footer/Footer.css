/* Footer */
footer {
	background-color: var(--white);
	color: var(--text-color);
	padding: 1.5rem;
	text-align: center;
}

footer .social-icons {
	margin: 1rem 0;
}

.social-icons img {
	cursor: pointer;
}

.social-icons img:hover {
	filter: invert(50%) sepia(90%) saturate(1000%) hue-rotate(350deg)
		brightness(95%) contrast(90%);
}

footer .social-icons a:hover {
	color: var(--secondary-color);
}
