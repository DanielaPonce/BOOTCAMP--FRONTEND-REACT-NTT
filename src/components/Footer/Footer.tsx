import './Footer.css'

const Footer = () => {
	return (
		<footer>
			<p>Síguenos en nuestras redes sociales</p>
			<div className="social-icons">
				<img src="/src//assets/icons/fb-icon.svg" alt="facebook icon" />
				<img
					src="/src/assets/icons/instagram-icon.svg"
					alt="instagram icon"
				/>
				<img
					src="/src/assets/icons/twitter-icon.svg"
					alt="twitter icon"
				/>
			</div>
			<p>© 2024 My Market. Todos los derechos reservados.</p>
			<p>
				<a href="#">Política de privacidad</a> |
				<a href="#">Términos de uso</a> | <a href="#">Contacto</a>
			</p>
		</footer>
	);
};

export default Footer;
