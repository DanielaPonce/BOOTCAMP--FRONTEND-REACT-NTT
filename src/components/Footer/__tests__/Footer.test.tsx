import { render } from '@testing-library/react';
import Footer from '../Footer';

describe('Footer Component', () => {
	test('matches snapshot', () => {
		const { container } = render(<Footer />);
		expect(container).toMatchSnapshot();
	});
});
