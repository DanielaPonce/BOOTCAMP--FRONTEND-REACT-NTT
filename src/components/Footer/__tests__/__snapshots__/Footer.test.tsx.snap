// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component matches snapshot 1`] = `
<div>
  <footer>
    <p>
      Síguenos en nuestras redes sociales
    </p>
    <div
      class="social-icons"
    >
      <img
        alt="facebook icon"
        src="/src//assets/icons/fb-icon.svg"
      />
      <img
        alt="instagram icon"
        src="/src/assets/icons/instagram-icon.svg"
      />
      <img
        alt="twitter icon"
        src="/src/assets/icons/twitter-icon.svg"
      />
    </div>
    <p>
      © 2024 My Market. Todos los derechos reservados.
    </p>
    <p>
      <a
        href="#"
      >
        Política de privacidad
      </a>
       |
      <a
        href="#"
      >
        Términos de uso
      </a>
       | 
      <a
        href="#"
      >
        Contacto
      </a>
    </p>
  </footer>
</div>
`;
